web/portal/client/src/entities/HolidayDate/Action/ImportHolidayDates.tsx

```diff
@@ -0,0 +1,108 @@
+import { DropdownChoices } from '@irontec/ivoz-ui';
+import { StyledTable } from '@irontec/ivoz-ui/components/List/Content/Table/ContentTable.styles';
+import { StyledDropdown } from '@irontec/ivoz-ui/services/form/Field/Dropdown/Dropdown.styles';
+import _ from '@irontec/ivoz-ui/services/translations/translate';
+import {
+  Checkbox,
+  SelectChangeEvent,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+
+type ImportHolidayDatesMappingTablePropsType = {
+  csv: Array<string>[];
+  columns: string[];
+  setColumns: (value: string[]) => void;
+  ignoreFirstLine: boolean;
+  setIgnoreFirstLine: (value: boolean) => void;
+};
+
+const ImportHolidayDatesMappingTable = (
+  props: ImportHolidayDatesMappingTablePropsType
+): JSX.Element => {
+  const { columns, setColumns, csv, ignoreFirstLine, setIgnoreFirstLine } =
+    props;
+
+  const choices: DropdownChoices = [
+    { id: 'name', label: 'Name' },
+    { id: 'eventDate', label: 'Event data' },
+  ];
+
+  const changeColumnMappingHandler = (event: SelectChangeEvent) => {
+    const { name, value } = event.target;
+    const newVal = [...columns];
+    newVal[parseInt(name, 10)] = value;
+    setColumns(newVal);
+  };
+
+  const changeIgnoreFirstLineHandler = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    checked: boolean
+  ) => {
+    setIgnoreFirstLine(checked);
+  };
+
+  return (
+    <>
+      <StyledTable size='medium' sx={{ overflowY: 'auto' }}>
+        <TableHead>
+          <TableRow>
+            {csv[0].map((row, key) => {
```

Comment:
ignore row here if you are not using it
```suggestion
            {csv[0].map((_, key) => {
```